sdk/servicebus/service-bus/test/internal/batchReceiver.spec.ts

```diff
@@ -807,37 +814,15 @@ describe("Batching Receiver", () => {
   });
 
   describe("Batch Receiver - disconnects", () => {
-    function simulateDisconnectDuringDrain(
-      receiverContext: ConnectionContext,
-      batchingReceiver: LinkEntity<Receiver> | undefined,
-      didRequestDrainResolver: Function
-    ) {
-      if (!batchingReceiver || !batchingReceiver.isOpen()) {
-        throw new Error(`batchingReceiver is not open or passed undefined.`);
-      }
-      // We want to simulate a disconnect once the batching receiver is draining.
-      // We can detect when the receiver enters a draining state when `addCredit` is
-      // called while didRequestDrainResolver is called to resolve the promise.
-      const addCredit = batchingReceiver["link"]!.addCredit;
-      batchingReceiver["link"]!.addCredit = function(credits) {
-        // This makes sure the receiveMessages doesn't end because of draining before the disconnect is triggered
-        // Meaning.. the "resolving the messages" can only happen through the onDetached triggered by disconnect
-        batchingReceiver["link"]!.removeAllListeners(ReceiverEvents.receiverDrained);
-        addCredit.call(this, credits);
-        if (batchingReceiver["link"]!.drain) {
-          didRequestDrainResolver();
-          // Simulate a disconnect being called with a non-retryable error.
-          receiverContext.connection["_connection"].idle();
-        }
-      };
-    }
-
-    describe(noSessionTestClientType + ": Batch Receiver - disconnects", function(): void {
+    describe("Batch Receiver - disconnects (non-session)", function(): void {
       before(() => {
+        enableCommonLoggers();
+        console.log(`Entity type: ${noSessionTestClientType}`);
```

Comment:
Is this getting removed?

Reply:
Oh, this was annoying. So if you have a template literal or a string concat then VS Code won't show the describe(<testname>) for it (it just shows up as describe(callback) instead. 

The test names here are basically the same so I was constantly getting confused. So I just made the test name a constant and then console.log this instead.